STREAMING/SwitchInPlayer.md

```diff
@@ -1,6 +1,5 @@
 ---
 ns: STREAMING
-aliases: ["0xD8295AF639FD9CB8"]
```

Comment:
Why remove this?

Reply:
He added it, then I told him to remove it as hex's are implied as being an alias (as no other item has a hex as the alias), it wasn't there originally.